src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        Figure [] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+        figures[0] = figureSupplier.getRandomFigure();
+        figures[1] = figureSupplier.getRandomFigure();
+        figures[2] = figureSupplier.getRandomFigure();
+        figures[3] = figureSupplier.getDefaultFigure();
+        figures[4] = new Rectangle("GREEN", 3,6);
+        figures[5] = new IsoscelesTrapezoid("ORANGE", 9, 5,4);

```

Comment:
According to task: The other half of the figures should have the same, default parameters.
You shouldn't put some predefined values to constructor, use figureSupplier.getDefaultFigure() for whole second part of the aarray